utils/g.html2man/ggroff.py

```diff
@@ -131,7 +131,7 @@ def fmt(self, format, content, var=None):
 
     def pp_li(self, content):
         if self.get("in_ul"):
-            self.fmt("\n.IP \(bu 4n\n@", content)
+            self.fmt(r"\n.IP \(bu 4n\n@", content)
```

Comment:
These are not the same:

```python
>>> "\n.IP \(bu 4n\n@"
'\n.IP \\(bu 4n\n@'
>>> r"\n.IP \(bu 4n\n@"
'\\n.IP \\(bu 4n\\n@'
```

Reply:
Try:

```python
>>> "\n.IP \\(bu 4n\n@"
'\n.IP \\(bu 4n\n@'
```

Reply:
This line of code has been changed as follows. 

 ```python
self.fmt("\n.IP \\(bu 4n\n@", content)
```